cloud/amazon/ec2.py

```diff
@@ -212,7 +212,7 @@
     required: false
     default: 'present'
     aliases: []
-    choices: ['present', 'absent', 'running', 'stopped']
+    choices: ['present', 'absent', 'running', 'rebooted', 'stopped']
```

Comment:
Hrm, that's tough. In my experience, I rarely need to do a stop/start of an instance in one go, so maybe we could use "restarted" for this, and then document that for a stop/start users should do:

```
- ec2: <some options> state=stopped
- ec2: <some options> state=running
```
```